apps/remote_control/lib/lexical/remote_control/build.ex

```diff
@@ -16,10 +16,7 @@ defmodule Lexical.RemoteControl.Build do
   end
 
   def compile_source_file(%Project{} = project, %SourceFile{} = source_file) do
-    # Don't format mix.exs files until we come up with with a way to prevent them from causing errors
-    unless Path.absname(source_file.path) == "mix.exs" do
-      RemoteControl.call(project, GenServer, :cast, [__MODULE__, {:compile_file, source_file}])
-    end
+    RemoteControl.call(project, GenServer, :cast, [__MODULE__, {:compile_file, source_file}])
```

Comment:
did you fix this? Prior, this broke pretty hard when you'd edit the `mix.exs` in the root directory, and it applied to every `mix.exs`, not just lexical's.

Reply:
hm, looks like this was already fixed and unrelated to this PR. Removing this `unless` from the `main` branch doesn't cause any issues when formatting `mix.exs` files.

Ok to leave in the PR?

Reply:
sure, we can always add it back.